### DISCUSSION TITLE
Commenting on a breakpoint does "spooky action at a distance"

### DISCUSSION THREAD

**Discussion by jazzdan** [RecordReplay/devtools#3216] [Q&A] [answered]
Clicking the comment button on a breakpoint didn't do what I expected

https://www.loom.com/share/d32a28f34a3a4ea6a091e79d81308985

I expected something to happen where I clicked the button, not off to the left in the sidebar

**Comment by jasonLaster** [comment]
@jonbell-lot23 this is something that we're currently discussing around comments

**Comment by jonbell-lot23** [comment, marked as answer]
We agree that this causes confusion sometimes but it's the pattern we've settled on for now.
